examples/ping.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"github.com/intel-go/nff-go/flow"
+	"flag"
+)
+
+func main() {
+	inport := flag.Uint("inport", 0, "port for receiver")
+        flag.Parse()
+
+	flow.SystemInit(&flow.Config{})
+
+	inputFlow, _ := flow.SetReceiver(uint16(*inport))
+	flow.SetIPForPort(uint16(*inport), uint32(20)<<24|uint32(20)<<16|uint32(20)<<8|uint32(20))
```

Comment:
You can use function packet.BytesToIPv4 and I think it is better to configure this address on command line.